definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -9301,5 +9301,1979 @@
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "SolStatementOfReconciliationFirm",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "SolStatementOfReconciliationName",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "StatementOfReconciliationComments",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "31/08/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "PreviousIssueDate",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CaseHistory",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespLinkedToCase",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespLinkedToCaseDate",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ReceivedAnswerFromCoResp",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ReceivedAnswerFromCoRespDate",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ReceivedAosFromCoResp",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ReceivedAosFromCoRespDate",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "respondentSolicitorRepresented",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "D8caseReferenceLabel",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/04/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DateAndTimeOfHearing",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/04/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DecreeNisiGrantedDate",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DNApprovalDate",
```

Comment:
It appears this is duplicated in the file

Reply:
removed

Reply:
Did you reintroduce it by accident?
https://github.com/hmcts/div-ccd-definitions/blob/017f89f11ec841169d4edd46ec3f0b4ce0c7158e/definitions/divorce/json/AuthorisationCaseField.json#L7229
Or am I looking at an old commit.